src/test/java/org/aksw/simba/squirrel/fetcher/http/HTTPFetcherTest.java

```diff
@@ -0,0 +1,24 @@
+package org.aksw.simba.squirrel.fetcher.http;
+
+
+import org.aksw.simba.squirrel.data.uri.CrawleableUri;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URI;
+
+import static org.junit.Assert.assertNotNull;
+
+public class HTTPFetcherTest {
+
+
+    @Test
+    public void fetch() throws Exception{
+        CrawleableUri tempUri = new CrawleableUri(new URI("http://danbri.org/foaf.rdf"));
+        HTTPFetcher fetcher = new HTTPFetcher();
+        File data = fetcher.fetch(tempUri);
+        assertNotNull(data);
```

Comment:
The problem with this test is that it is a) depending on an available internet connection and b) depending on a resource which is not controller by us. Please take a look at https://github.com/varunmaitreya/Squirrel/blob/feature_Integration_test_case/src/test/java/org/aksw/simba/squirrel/simulation/ScenarioBasedTest.java to see how you can set up a local HTTP server in an easy way.

Reply:
@MichaelRoeder Have setup a local HTTP server connection and made httpfetcher to run on the local server.